Add More Realistic Weapon Weight

charabaruk opened the pull request:
* Add plugins
  - To Gameplay - Economy & Item Balance section

* Add link
  - https://www.nexusmods.com/oblivion/mods/47321/

charabaruk committed:
Add More Realistic Weapon Weight

sibir-ine reviewed (changes requested):
A location isn't enough metadata to justify adding a plugin. You could add cleaning data (or Bash Tags from the sounds of it, but you should verify with xEdit & [WB's docs](https://wrye-bash.github.io/docs/Wrye%20Bash%20Advanced%20Readme.html#patch-tags)).

charabaruk commented:
Yeah I don't think clean/dirty matters for these, but good call on the Bash tags.

charabaruk committed:
Add Bash tags for More Realistic Weapon Weight

Same tags (Filter, Stats) applies for all plugins in the mod. It's a
shame that we can't specify exactly /what/ stat to merge from these ESPs
(WEAP.DATA.Weight).

charabaruk commented:
Tags added.

sibir-ine commented on masterlist.yaml:
```diff
@@ -9965,6 +9965,14 @@ plugins:
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/47321/'
+        name: 'More Realistic Weapon Weight'
+    tag:
+      - Filter
+      - Stats
```
```suggestion
    tag:
      - Deactivate
      - NoMerge
      - Stats
  - name: 'AllDlC''s-WeaponWeight.esp'
    tag: [ Filter ]
```

**Filter** only makes sense for the DLC plugin as all of the masters for the other plugins are always required for each. **Deactivate** & **NoMerge** would allow only the stats changes of each plugin to be merged.

charabaruk committed:
Update More Realistic Weapon Weight tags

charabaruk commented:
Tags updated.
